lblod/app-lokaal-mandatenbeheer
# Ben/lmb 525 yeet leidinggevenden

## Description

Remove leidinggevenden logic.

## How to test

Everything should still work as before. Probably best to just remove your database, restart your docker containers and just see if the app still works as expected.

## Links to other PR's

- https://github.com/lblod/frontend-lokaal-mandatenbeheer/pull/251
